@here/harp-geoutils/lib/projection/TransverseMercatorProjection.ts

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { GeoBox } from "../coordinates/GeoBox";
+import { GeoCoordinates } from "../coordinates/GeoCoordinates";
+import { GeoCoordinatesLike } from "../coordinates/GeoCoordinatesLike";
+import { Box3Like, isBox3Like } from "../math/Box3Like";
+import { MathUtils } from "../math/MathUtils";
+import { isOrientedBox3Like, OrientedBox3Like } from "../math/OrientedBox3Like";
+import { Vector3Like } from "../math/Vector3Like";
+import { EarthConstants } from "./EarthConstants";
+import { Projection, ProjectionType } from "./Projection";
+
+/**
+ *
+ * https://en.wikipedia.org/wiki/Transverse_Mercator_projection
+ * http://mathworld.wolfram.com/MercatorProjection.html
+ *
+ */
+export class TransverseMercatorProjection extends Projection {
```

Comment:
No need to export this class

Reply:
Ok

Reply:
Actually, how it would be best to do with the case when i need some method special to transverse Mercator?

https://github.com/heremaps/harp.gl/pull/653/files#diff-1b0ff12045051a049d60bf7d1c58e4a3R186

Here i cast to TransverseMercatorProjection explicitly just to use this method. If i don't export that class, i will need to cast to any.

Reply:
as far as I can see you are using alignLongitude but this method could actually be a free function in this file or you could also move it to the PolarTileDataSource